src/components/side_panel/find_and_replace/find_and_replace_store.ts

```diff
@@ -260,6 +263,7 @@ export class FindAndReplaceStore extends SpreadsheetStore implements HighlightPr
 
     // Switch to the sheet where the match is located
     if (jumpToMatchSheet && this.getters.getActiveSheetId() !== selectedMatch.sheetId) {
```

Comment:
is `jumpToMatchSheet` still needed ? Wouldn't it work with something like:
```suggestion
    if (!this.isAutoSheetSwitch && this.getters.getActiveSheetId() !== selectedMatch.sheetId) {
```